app/src/components/games/crescendo/CrescendoAutoPath.ts

```diff
@@ -1,4 +1,4 @@
-import {CrescendoActionType} from './CrescendoActions';
+import {CrescendoActionType} from './CrescendoActions.tsx';
```

Comment:
why?

Reply:
also remember to reformat file so that it is consistant formatting-wise with spacing after the braces

Reply:
quotation marks too